delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -170,6 +170,8 @@ public void initialize(WorkerContext context) throws Exception {
     Set<DDLOperation> ddlBlacklist = new HashSet<>(config.getDdlBlacklist());
     // targets will not behave properly if they don't get create table events
     ddlBlacklist.remove(DDLOperation.CREATE_TABLE);
+    // blacklist drop database event by default in case that user unintentionally drop database from source
+    ddlBlacklist.add(DDLOperation.DROP_DATABASE);
```

Comment:
instead of adding it here, where the user can never change the behavior, it's better to change config.getDdlBlacklist() to return a list with DROP_DATABASE if the value is null. That way the user can overwrite it if they wish.